### PR TITLE
Styling ready to claim section.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Styles the 'Ready to claim?' section by adding a FAQ drawer component.
- Styles the "Important!" message below. Intend to use the same component for other user messages (potentially incl. the validation errors).

Desktop

https://user-images.githubusercontent.com/31534717/151393568-4ac5d7f1-1c91-4e82-b457-60a4e2521bac.mov

Mobile

https://user-images.githubusercontent.com/31534717/151393624-fc29ca16-958a-47b8-8c9e-41fd16406d5b.mov

### CODE REVIEW

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 1]
```diff
@@ -55,6 +58,19 @@ const STEPS_DATA = [
...
 ]
 
+const FAQ_DATA = [
+  {
+    title: 'What will happen?',
+    content:
+      'By sending this Ethereum transaction, you will be investing tokens from the connected account and exchanging them for vCOW tokens that will be received by the claiming account specified above.',
+  },
+  {
+    title: 'Can I modify (partial) invested amounts later?',
```
Comment (biocom) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:68]: partially?

**Review by W3stside — APPROVED** [review]
nice! probably some iterations we can make on wording but looks good

**Review by alfetopito — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @biocom , seems like 'Important' section's color is a bit pale and hard to read in the dark mode. WDYT?
image

Then, in real mobile devices (iOS and Android) the cross icon stays focused when close a question: 
image

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 2]
```diff
@@ -271,18 +287,14 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
...
-          <p>
-            <b>Can I modify the invested amounts or invest partial amounts later?</b> No. Once you send the transaction,
-            you cannot increase or reduce the investment. Investment opportunities can only be exercised once.
-          </p>
-          <p>
-            <b>Important!</b> Please make sure you intend to claim and send vCOW to the mentioned receiving account(s).
-          </p>
+          <FaqDrawer items={FAQ_DATA} />
+          <UserMessage>
+            <SVG src={ImportantIcon} description="Important!" />
```
[the remark this answers is not in the document]
Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx:292]: Yes, will review and tweak.

**Comment by biocom** [comment]
@elena-zh 

> seems like 'Important' section's color is a bit pale and hard to read in the dark mode. WDYT?

Looking at your screenshot indeed 🤔 . Will review.

> Then, in real mobile devices (iOS and Android) the cross icon stays focused when close a question:

This is an hover effect. So given on mobile that translates to 'on touch' that's likely why it remains active.

**Comment by biocom** [comment]
@elena-zh @W3stside Addressed both your points:
Screen Shot 2022-01-27 at 18 05 45
Screen Shot 2022-01-27 at 18 05 38

File: src/custom/pages/Claim/InvestmentFlow/index.tsx (hunk 1 above)
Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx]: Updated to use `partially`

File: src/custom/components/FaqDrawer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+`
+
+type FaqDrawerProps = {
+  items: {
+    title: string
+    content: string
+  }[]
+}
+
+export function FaqDrawer({ items }: FaqDrawerProps) {
```
Comment (anxolin) [inline src/custom/components/FaqDrawer/index.tsx:79]: Awesome you took the chance to do a reusable component ❤️

**Review by anxolin — APPROVED** [review]
Looks really great! 
I have some heasitation with the red color being to strong, but looks good.

Not from this PR, but if we go for "scary message", shouldn't we mention also that the person will also be investing, or is it  too repetitive? 
image

**Comment by biocom** [comment]
> Not from this PR, but if we go for "scary message", shouldn't we mention also that the person will also be investing, or is it too repetitive?

Yes I think that makes sense, given you only land on this page when you're investing. Should we rename to:

> Important! Please make sure you intend to claim/invest in vCOW and send to the above mentioned receiving account.

**Comment by elena-zh** [comment]
LTM now!

**Pull request merged**